plugins/generator-1.16.4/forge-1.16.4/procedures/world_data_isremote.java.ftl

```diff
@@ -0,0 +1 @@
+((world instanceof World)?((World) world).isRemote:false)
```

Comment:
Better 'world.isRemote()'. Cleaner an no casting.

Reply:
So i need to replace the line with this? `((world instanceof World)?(world.isRemote():false)`

Reply:
No, just what I said, I said you don't need casting, why would you cast if isRemote() methods is part of IWorld?

Reply:
Oh okay, I see...

Reply:
Done!